bench/wtperf/runners/500m-btree-rdonly.wtperf

```diff
@@ -17,5 +17,5 @@ report_interval=10
 run_time=7200
 sample_interval=10
 sample_rate=1
-threads=((count=20,reads=1))
+threads=((count=20,reads=1,throttle=10000))
```

Comment:
I'm a little surprised that this is the right throttle config - I thought the number was applied to each thread, so 10k would result in 20x10k ops/second. If that isn't how it works we should update the wtperf configuration option description to match reality (https://github.com/wiredtiger/wiredtiger/blob/develop/bench/wtperf/wtperf_opt.i#L192)

Reply:
My mistake. I misunderstood this configuration and have fixed it now to be divided by the number of threads. I ran it just then with 500 for 12 min and got:
```
Executed 7196882 read operations (100%) 9995 ops/sec
```